### ISSUE TITLE
ACF Pro 5.7.0 beta – Media Modal not working with Gutenberg editor

### ISSUE THREAD

**Issue by VeVas** [AdvancedCustomFields/acf#34]
Just testet the new ACF Pro 5.7.0 beta version for a couple of days, and the Media Modal won't open on image fields in Gutenberg editor. Nothing happens if you click the "Add media"-button. Also not able do edit images already saved on posts created with previous version of ACF. This was working fine with ACF Pro 5.6.10, and same Gutenberg version (2.7.0)

WordPress 4.9.5
ACF PRO 5.7.0 (beta 2 & 3)
Gutenberg 2.7.0

**Comment by elliotcondon** [comment]
Hi @VeVas 

Thanks for the bug report. I'll perform some testing and get back to you shortly

**Comment by elliotcondon** [comment]
I've just done some testing, but can't replicate the issue.
Can you please check your console log for any JS errors?

I did notice a different issue (much larger) which is no click event is working on ACF fields!
This is most likely due to some new JS added in Gutenberg to capture all click events and prevent the listeners.

I'll flag this with core if not already fixed.

**Comment by VeVas** [comment]
Hmm, ok. This is the console output:

ACF Pro 5.6.10 (Working):
![skjermskot-2018-04-30-kl -08 46 17](https://user-images.githubusercontent.com/7713087/39417272-65664498-4c53-11e8-8170-a5838963e2d6.jpg)

ACF PRO 5.7.0-beta3 (Not working):
![skjermskot-2018-04-30-kl -08 45 14](https://user-images.githubusercontent.com/7713087/39417276-696061e6-4c53-11e8-8bc6-7bdbd91d5857.jpg)

Let me know, and I can send you the log files :)

**Comment by elliotcondon** [comment]
Thanks for the reply.

Can you replicate this on a fresh WP setup without any other plugins / theme installed?
If so, please also export your field group to .json and attach to your reply. I'll import the same fields and test again.

**Comment by VeVas** [comment]
Same issues with fresh install. See json-code below. I'm in my local dev environment, but can try on a live server. 

`[
    {
        "key": "group_5ae6d348313c9",
        "title": "test_group",
        "fields": [
            {
                "key": "field_5ae6d34f1241c",
                "label": "image",
                "name": "image",
                "type": "image",
                "instructions": "",
                "required": 0,
                "conditional_logic": 0,
                "wrapper": {
                    "width": "",
                    "class": "",
                    "id": ""
                },
                "return_format": "array",
                "preview_size": "thumbnail",
                "library": "all",
                "min_width": "",
                "min_height": "",
                "min_size": "",
                "max_width": "",
                "max_height": "",
                "max_size": "",
                "mime_types": ""
            },
            {
                "key": "field_5ae6d3571241d",
                "label": "file",
                "name": "file",
                "type": "file",
                "instructions": "",
                "required": 0,
                "conditional_logic": 0,
                "wrapper": {
                    "width": "",
                    "class": "",
                    "id": ""
                },
                "return_format": "array",
                "library": "all",
                "min_size": "",
                "max_size": "",
                "mime_types": ""
            },
            {
                "key": "field_5ae6d35f1241e",
                "label": "gallery",
                "name": "gallery",
                "type": "gallery",
                "instructions": "",
                "required": 0,
                "conditional_logic": 0,
                "wrapper": {
                    "width": "",
                    "class": "",
                    "id": ""
                },
                "min": "",
                "max": "",
                "insert": "append",
                "library": "all",
                "min_width": "",
                "min_height": "",
                "min_size": "",
                "max_width": "",
                "max_height": "",
                "max_size": "",
                "mime_types": ""
            }
        ],
        "location": [
            [
                {
                    "param": "post_type",
                    "operator": "==",
                    "value": "page"
                }
            ]
        ],
        "menu_order": 0,
        "position": "normal",
        "style": "default",
        "label_placement": "top",
        "instruction_placement": "label",
        "hide_on_screen": "",
        "active": 1,
        "description": ""
    }
]`

**Comment by joshuafredrickson** [comment]
I'm seeing the same thing on a fresh install using 5.7.0-beta3. Media gallery modals don't open and also don't trigger any console warnings.

Works fine (but with hook name warnings) in 5.6.10.

**Comment by elliotcondon** [comment]
Hi guys,

Thanks for the updates. This appears to be fixed in the latest build (beta4) which will be released later this week!

**Comment by andremacola** [comment]
Some errors are fixed in beta4 but not all of them. See image below with some 404 .css problems.

![captura de tela 2018-05-18 as 14 44 20](https://user-images.githubusercontent.com/3408809/40249666-113b90ca-5aaa-11e8-9d19-ab1538dc4529.png)

**Comment by elliotcondon** [comment]
Hi @andremacola 

Thanks for the reply. The errors you have reported are coming from .css files outside of the ACF PRO plugin. Are you sure these issues are related to ACF PRO?

**Comment by andremacola** [comment]
@elliotcondon well, no errors when deactivating ACF PRO and another problem is occurring with me is field previously saved at _sidebar location_ (5.6 field), when changing for _after content_, nothing happens (always using gutenberg). I need to duplicate the group. This happened to me twice.

**Comment by andremacola** [comment]
@elliotcondon  I think this is something with **Editor** and **Gallery Field**

The Editor Field is blank at Gutenberg

![captura de tela 2018-05-18 as 22 40 36](https://user-images.githubusercontent.com/3408809/40263685-862db556-5aec-11e8-909a-06e3502e8a17.png)

**Comment by elliotcondon** [comment]
Thanks @andremacola 

I've just added this to my to-do list and will be sure to test this thoroughly over the next few weeks as we shift focus to Gutenberg testing.

**Comment by andremacola** [comment]
@VeVas did you try all the ACF Fields? Especially the Editor and Gallery fields?

![1](https://user-images.githubusercontent.com/3408809/40393504-1501de54-5df6-11e8-956d-0248f2adadfc.png)
![captura de tela 2018-05-22 as 19 24 26](https://user-images.githubusercontent.com/3408809/40393507-16c89534-5df6-11e8-8f3e-7792814f9450.png)
![captura de tela 2018-05-22 as 19 24 36](https://user-images.githubusercontent.com/3408809/40393509-185d29b4-5df6-11e8-990c-991313abb33c.png)

**Comment by VeVas** [comment]
@andremacola Yes, I checked all the fields using the media modal. But I just noticed that the editor is not working when I create a new page, only on existing pages. Seems to be related with a lot of treads in the Gutenberg issues regarding loading of TinyMCE-versions.
